api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -500,6 +500,13 @@ message ClusterManager {
   // <envoy_v3_api_enum_value_config.core.v3.ApiConfigSource.ApiType.GRPC>`.
   core.v3.ApiConfigSource load_stats_config = 4;
 
+  // If set Envoy will use the :ref:`ads_config
+  // <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.DynamicResources.ads_config>`
+  // config source as the server and use the same connection.
+  // If ``ads_config`` is not defined the bootstrap will be rejected.
+  // Cannot be used if ``load_stats_config`` is defined.
+  bool load_stats_over_ads_config_connection = 6;
```

Comment:
Since both ADS and ClusterManager are part of the bootstrap, is setting `load_stats_over_ads_config_connection` to true equivalent to making `load_stats_config` a copy of `ads_config` ?

Reply:
It is currently not supported by Envoy.
The server will need to send a different config, depending on the client that connects to it.